catalyst/dl/callbacks/mixup.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+
+import numpy as np
+
+import torch
+
+from catalyst.dl.core.state import RunnerState
+from catalyst.dl.callbacks import CriterionCallback
+
+
+class MixupCallback(CriterionCallback):
+    """
+    Callback to do mixup augmentation.
+
+    Paper: https://arxiv.org/abs/1710.09412
+
+    Note:
+        MixupCallback is inherited from CriterionCallback and
+        does its work.
+
+        You may not use them together.
+    """
+    def __init__(self, fields: List[str] = ("features",), alpha=1.0,
+                 train_only=True, **kwargs):
```

Comment:
could you please refactor to 
```
def __init__(
    self, 
    fields: List[str] = ("features",), 
    alpha=1.0,
    train_only=True, 
    **kwargs
):
```
one more common lib standard :)

Reply:
Done :)